helpers/hashPassword.js

```diff
@@ -0,0 +1,9 @@
+const bcrypt = require('bcryptjs');
+
+
+
```

Comment:
There seems to be a lot of extra spacing here. Try out [prettier](https://prettier.io) if you want a code formatting tool to take care of stuff like this for you! It's even more common if you're working on code that more people than just you will touch, so it's a good thing to get used to using.